pkg/env/list.go

```diff
@@ -19,4 +19,8 @@ var (
 	// This is ignored if SkipPeerValidation is enabled.
 	// This variable was copied over from the stackrox repo.
 	OpenshiftAPI = RegisterBooleanSetting("ROX_OPENSHIFT_API", false)
+
+	// NodeName is used when running Scanner in Node Inventory mode. This should be set by
+	// Kubernetes in the Secured Cluster.
+	NodeName = RegisterSetting("ROX_NODE_NAME", WithDefault("unset"))
 )
```

Comment:
I agree with that. Had a chat with Piotr this morning and we think it's better to crash it with an error message. Because if the node name isn't set, or it's set to a dummy value, Sensor will not be able to link the inventory message with the v1.Node message that comes from K8s. RHCOS will *silently fail*, and users will have to look through the logs to figure out that an env was not set. 

For that reason, it makes more sense to fail fast and crash node inventory with an error message saying that we need `ROX_NODE_NAME` to be set to an unique node value, likely coming from Kubernetes.